pkg/simple/client/s2is3/options.go

```diff
@@ -18,14 +18,14 @@ type S3Options struct {
 
 func NewS3Options() *S3Options {
 	return &S3Options{
-		Endpoint:        "",
-		Region:          "",
+		Endpoint:        "http://openpitrix-minio.openpitrix-system.svc:9000",
```

Comment:
Enpoint/host/apiserver fields are left empty for purpose, since we treat components as disabled if these fields was empty. If we set default values for these fields, means enable them by default